src/mpyl/projects/versioning.py

```diff
@@ -113,8 +113,8 @@ def upgrade(self, previous_dict: ordereddict) -> ordereddict:
         return previous_dict
 
 
-class ProjectUpgraderOneFour20(Upgrader):
-    target_version = "1.4.20"
+class ProjectUpgraderOneSix7(Upgrader):
```

Comment:
I'm confused, why are you replacing the 4.20 upgrader instead of adding a new one?